### PR TITLE
Api metric panel

### PR DESCRIPTION
**Pull request by Dominic-Marcelino**

### CODE REVIEW

File: packages/api-metric-panel/src/api-metric-panel/panel.vue [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+
+onBeforeUnmount(() => {
+	unmountResizeObserver();
+	unsubscribeInsightsStore();
+});
+
+
+function displayValue(value: MetricType) {
+	if (value === null || value === undefined) {
+		return '...Loading';
```
Comment (formfcw) [inline packages/api-metric-panel/src/api-metric-panel/panel.vue:200]: Could use `t('loading')` instead. (from `const { t } = useI18n();`)

**Review by formfcw — COMMENTED** [review]
Great work @Dominic-Marcelino 🎉 

Please be sure to update the code to at least fix the bug with operator choices (condition for disabled) as well as the bug where the panel value does not update as expected.

File: packages/api-metric-panel/package.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+				"name": "api-metric-endpoint",
+				"source": "src/api-metric-endpoint/index.ts"
+			},
+			{
+				"type": "panel",
+				"name": "api-metric-panel",
+				"source": "src/api-metric-panel/index.ts"
+			}
+		],
+		"host": "^10.10.0"
```
[the remark this answers is not in the document]
Comment (Dominic-Marcelino) [reply packages/api-metric-panel/package.json:41]: Didn't knew this was possible - good to know!

File: packages/api-metric-panel/src/api-metric-panel/panel.vue [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+
+onUpdated(() => {
+	updateFit();
+});
+
+const unsubscribeInsightsStore = insightsStore.$onAction(
+  ({name, store, args, after, onError,}) => {
+		if (name === 'refresh') {
+			fetchMetric();
+		}
```
[the remark this answers is not in the document]
Comment (Dominic-Marcelino) [reply packages/api-metric-panel/src/api-metric-panel/panel.vue:188]: For doumentation purpose: `stagePanelUpdate` won't work, as the value isn't updated ad that point.

**Comment by formfcw** [comment]
Thanks a lot 🚀 LGTM!

**Review by phazonoverload — APPROVED** [review]

**Pull request merged**